File: Behavorial/Command/Command_Python/Radio.py
from ElectronicDevice import *

class Radio(ElectronicDevice):

    def __init__(self):
        self.volume = 0
        self.name = 'Radio'

    def on(self):
        print('TV On')

    def volumeUp(self):
        self.volume += 1
        print('Radio Volume : ' + str(self.volume))

    def off(self):
        print('TV Off')

    def volumeDown(self):
        self.volume -= 1
        print('Radio Volume : ' + str(self.volume))

File: Behavorial/Command/Command_Java/Radio.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Command;

/**
 *
 * @author shantanu1395
 */

//Receiver
public class Radio implements ElectronicDevice{
    
    private int volume = 0;
    
    @Override
    public void on() {
        System.out.println("Radio On");
    }

    @Override
    public void off() {
        System.out.println("Radio Off");
    }

    @Override
    public void volumeUp() {
        volume++;
        System.out.println("Radio Volume :"+volume);
    }

    @Override
    public void volumeDown() {
        volume--;
        System.out.println("Radio Volume :"+volume);
    }
}
